app/models/concerns/hyku_addons/account_behavior.rb

```diff
@@ -3,7 +3,9 @@
 module HykuAddons
   module AccountBehavior
     extend ActiveSupport::Concern
+    include ActiveModel::Validations
     included do
+      email_validation_format = URI::MailTo::EMAIL_REGEXP
```

Comment:
Do we need to store this in a variable?  It feels like it should either be a constant or skipped and use `URI::MailTo::EMAIL_REGEXP` directly in the `validates` below.

Reply:
Yeah you are probably right here.